drivers/dma/Kconfig.stm32

```
@@ -43,3 +43,21 @@ config DMA_STM32_SHARED_IRQS
 	  Enable shared IRQ support on devices where channels share 1 IRQ.
 
 endif # DMA_STM32
+
+
+config BDMA_STM32
+	bool "Enable STM32 BDMA driver"
+	select USE_STM32_LL_BDMA
+	depends on SOC_FAMILY_STM32
```

Only the H7 has an BDMA I think. Regardless, it's the only one it's implemented for.
```suggestion
	depends on SOC_SERIES_STM32H7X
```

---

Replaced